plugins/woocommerce/includes/legacy/class-wc-legacy-api.php

```diff
@@ -114,7 +114,12 @@ private function maybe_log_rest_api_request( string $route, ?string $user_agent
 		 * @since 8.5.0
 		 */
 		if ( apply_filters( 'woocommerce_log_legacy_rest_api_usages', true ) ) {
-			wc_get_logger()->info( 'Version: ' . WC_API_REQUEST_VERSION . ", Route: $route, User agent: $user_agent", array( 'source' => 'legacy_rest_api_usages' ) );
+            $info = 'Version: ' . WC_API_REQUEST_VERSION . ", Route: $route, User agent: $user_agent";
```

Comment:
While `$ip_address` is run through `sanitize_text_field( wp_unslash( ... ) )`, `$route` and even more so `$user_agent` are input from the user. We should probably do some basic cleaning before we write them to our log files?